documentation/smrt-versioning/docs/index.md

```diff
@@ -0,0 +1,4 @@
+# Smrt Versioning
```

Comment:
I think 'SMRT Link Versioning' makes more sense as the topic as SMRT covers a range of PacBio things.